src/examples/install-latest-chrome.yml

```diff
@@ -0,0 +1,11 @@
+description: >
+  Example job that installs the latest version of Google Chrome
+
+usage:
+  version: 2.1
+  orbs:
+    browser-tools: circleci/browser-tools@0.1.3
+  workflows:
+    main:
+      jobs:
+        - browser-tools/install-chrome
```

Comment:
There is no install-chrome job in this orb currently. There are only commands which could be utilized within a job.

https://circleci.com/docs/2.0/jobs-steps/